### DISCUSSION TITLE
How to disable title/halftitle/endpaper?

### DISCUSSION THREAD

**Discussion by jodros** [Omikhleia/resilient.sile#116] [Q&A] [answered]
I tried to do so by setting the options to none and false in `silm`, and also tried to run with template files empty, but just got about to five blank pages at the end!

**Comment by Omikhleia** [comment, marked as answer]
Just verified, and as far as I can tell the introductory text in the User Guide chapter 2 is correct.

So it works via inputters options, with `bookmatter=false` (and likewise `cover=false` to disable the cover, see §2.3)

```
sile -u inputters.silm[bookmatter=false] mybook.silm
```

As noted in the User Guide, the default setting depend on a property in your master document....
```
book:
  enabled: false|true
  # ... Any other cover or book matter setup you may need eventually
```

So you may have it to false (= it's default value actually), and explicitly use inputter options to enable the cover and or book matter pages; or set it to true and use inputter options to disable those which are not wanted.

I don't know what to say, it just seems to work as documented ^^ ;)

(And empty template files are legit: not all books have content on all these pages, they are just blank in some editions)

**Reply by Omikhleia** [reply]
Add-on note:
See also #193 for more flexibility eventually.
